### ISSUE TITLE
Can't kill running script after starting it from launch task

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/vscode-ev3dev-browser#57]
After launching a script which doesn't exit (e.g., by hitting <kbd>F5</kbd>)  the debug page shows an "in progress" scrolling bar:

![image](https://user-images.githubusercontent.com/3310349/34243855-ad773f54-e5d7-11e7-9128-6a1f47fccb40.png)

It never gets to the typical point of a debug session where the command menu appears along the top that would let you stop the debug session. Hence, I can't find a way to kill the script from VSCode.

Edit: For future readers, you can kill it by physically pressing the "back" button on the brick (unless you've reconfigured `brickrun`).

**Comment by dlech** [comment]
Can you please give more info about your environment. Host OS? ev3dev image version? etc.

**Comment by WasabiFan** [comment]
Windows 10. Build of extension from master.

```
========== ev3dev-sysinfo ==========
Image file:         ev3dev-stretch-ev3-generic-2017-09-30
Kernel version:     4.9.49-ev3dev-1.5.1-ev3
Brickman:           0.10.0
ev3devKit:          0.5.2
Board:              board0
BOARD_INFO_HW_REV=7
BOARD_INFO_MODEL=LEGO MINDSTORMS EV3
BOARD_INFO_ROM_REV=6
BOARD_INFO_SERIAL_NUM=00165340720B
BOARD_INFO_TYPE=main
```

**Comment by laurensvalk** [comment]
I can confirm this on Ubuntu 16.04, with ev3dev-browser 0.8.0 and vscode:

```
Version 1.19.2
Commit 490ef761b76b3f3b3832eff7a588aac891e5fe80
Date 2018-01-10T15:57:21.071Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

```
========== ev3dev-sysinfo ==========
Image file:         ev3dev-stretch-ev3-generic-2017-12-13
Kernel version:     4.9.58-ev3dev-1.6.0-ev3
Brickman:           0.10.0
ev3devKit:          0.5.2
Board:              board0
BOARD_INFO_HW_REV=7
BOARD_INFO_MODEL=LEGO MINDSTORMS EV3
BOARD_INFO_ROM_REV=6
BOARD_INFO_SERIAL_NUM=001653420B86
BOARD_INFO_TYPE=main
```

**Comment by laurensvalk** [comment]
This is fixed for me in ev3dev browser 0.8.1.

```
========== ev3dev-sysinfo ==========
Image file:         ev3dev-stretch-ev3-generic-2017-12-13
Kernel version:     4.9.58-ev3dev-1.6.0-ev3
Brickman:           0.10.0
BogoMIPS:           148.88
Board:              board0
BOARD_INFO_HW_REV=7
BOARD_INFO_MODEL=LEGO MINDSTORMS EV3
BOARD_INFO_ROM_REV=6
BOARD_INFO_SERIAL_NUM=0016533DDB29
BOARD_INFO_TYPE=main
```

**Comment by dlech** [comment]
Interesting. Must have been related to spaces in paths.
